mito-ai/src/Extensions/CellToolbarButtons/CellToolbarButtonsPlugin.tsx

```diff
@@ -38,6 +38,7 @@ const CellToolbarButtonsPlugin: JupyterFrontEndPlugin<void> = {
             },
             isVisible: () => notebookTracker.activeCell?.model.type === 'code' && app.commands.hasCommand(COMMAND_MITO_AI_SEND_EXPLAIN_CODE_MESSAGE)
         });
+        console.log("mito-ai: CellToolbarButtonsPlugin activated");
```

Comment:
Do we want to keep console logs here, or was this for debugging?

Reply:
I think it might be nice to have these, so when things aren't working right we can look at the console logs and see if all the extensions started up properly.